### ISSUE TITLE
[MAS2.4.4] [Screen reader -Endpoint] “Learn more ” link is not descriptive

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1915]
**Actual Result:**
“Learn more” link is not descriptive present in the Add an endpoint for your bot dialog 
Voiceover announced "Learn more" as "Learn more link".

**Expected Result:**
Learn more link should be more descriptive like Learn more about Azure for us government
For every link should more descriptive .

**Note:** 
The same issue fixed for Win32. Bug link: [https://github.com/microsoft/BotFramework-Emulator/issues/1795](url)

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Navigate to the Bot explorer and select it 
3. Navigate to the End point symbol control and select it 
4. Dialog gets opened 
5. Navigate to the Learn more link 
6. Observe the issue 

**User Impact:**
Users who rely on Screen Readers will be affected to navigate the links if Narrator is not narrating the descriptive info for link.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

**Attachment:**
[[MAS2.4.4] [Screen reader -Endpoint] “Learn more ” link is not descriptive.zip](https://github.com/microsoft/BotFramework-Emulator/files/3697018/MAS2.4.4.Screen.reader.-Endpoint.Learn.more.link.is.not.descriptive.zip)

**Comment by ShikhaMishra11** [comment]
Verified the issue and it is fixed in latest build.
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (version: 4.6.0-92058)
